db.json

```diff
@@ -2,27 +2,27 @@
   "companies": [
     {
       "id": 1,
-      "name": "Fusion LLC",
```

Comment:
In real life you usually don't want to change the backend data structures. Therefore, real life fix would include changing frontend code for this.

For dates, you would have to write a date converter on frontend.

For revenue numbers, you would have to write number formatter on frontend.

For company name you need to find a way to fix it on frontend, too.

Reply:
Wow, didn't know that I had to make changes on the front end. I am just starting out, thank you for the feedback. Will try to fix it